torch_geometric/datasets/fake.py

```diff
@@ -93,6 +89,9 @@ def generate_data(self) -> Data:
         elif self.edge_dim == 1:
             data.edge_weight = torch.rand(data.num_edges)
 
+        for feature_name, feature_shape in kwargs.items():
```

Comment:
```suggestion
        for feature_name, feature_shape in self.kwargs.items():
```
Let's use `kwargs` as a property (similar to all other variables).